### PR TITLE
Remove server configs

### PR DESCRIPTION
**Pull request by arthurvr**
Fixes https://github.com/yeoman/generator-gulp-webapp/issues/195.

### CODE REVIEW

**Comment by arthurvr** [comment]
Looks like test were already failing on master.

**Comment by austinpray** [comment]
:+1:

**Comment by silvenon** [comment]
Yeah, sorry about the tests, I have no idea why they are failing ATM.

File: app/templates/gulpfile.js [hunk 1]
```diff
@@ -63,7 +63,6 @@ gulp.task('extras', function () {
   return gulp.src([
     'app/*.*',
     '!app/*.html',
```
Comment (silvenon) [inline app/templates/gulpfile.js]: Please remove the trailing `,`.

Comment (arthurvr) [inline app/templates/gulpfile.js]: done

**Comment by silvenon** [comment]
Great, thanks :smiley:

**Pull request merged**